contracts/test/modules/ReserveAuction/V1/ReserveAuction.integration.t.sol

```diff
@@ -0,0 +1,219 @@
+
+        // Init V3
+        registrar.init(ZMM);
+        ZPFS.init(address(ZMM), address(0));
+
+        // Create users
+        seller = new Zorb(address(ZMM));
+        sellerFundsRecipient = new Zorb(address(ZMM));
+        bidder = new Zorb(address(ZMM));
+        otherBidder = new Zorb(address(ZMM));
+        finder = new Zorb(address(ZMM));
+        royaltyRecipient = new Zorb(address(ZMM));
+
+        // Deploy mocks
+        royaltyEngine = new RoyaltyEngine(address(royaltyRecipient));
+        token = new TestERC721();
+        weth = new WETH();
+
+        // Deploy Reserve Auction v1.0
+        auctions = new ReserveAuctionV1(
+            address(erc20TransferHelper),
+            address(erc721TransferHelper),
+            address(royaltyEngine),
+            address(ZPFS),
+            address(weth)
+        );
+        registrar.registerModule(address(auctions));
+
+        // Set balances
+        vm.deal(address(seller), 100 ether);
+        vm.deal(address(bidder), 100 ether);
+        vm.deal(address(otherBidder), 100 ether);
+
+        // Mint seller token
+        token.mint(address(seller), 0);
+
+        // Bidder swap 50 ETH <> 50 WETH
+        vm.prank(address(bidder));
+        weth.deposit{value: 50 ether}();
+
+        // otherBidder swap 50 ETH <> 50 WETH
+        vm.prank(address(otherBidder));
+        weth.deposit{value: 50 ether}();
+
+        // Users approve ReserveAuction module
+        seller.setApprovalForModule(address(auctions), true);
+        bidder.setApprovalForModule(address(auctions), true);
+        otherBidder.setApprovalForModule(address(auctions), true);
+
+        // Seller approve ERC721TransferHelper
+        vm.prank(address(seller));
+        token.setApprovalForAll(address(erc721TransferHelper), true);
+
+        // Bidder approve ERC20TransferHelper
+        vm.prank(address(bidder));
+        weth.approve(address(erc20TransferHelper), 50 ether);
+
+        // otherBidder approve ERC20TransferHelper
+        vm.prank(address(otherBidder));
+        weth.approve(address(erc20TransferHelper), 50 ether);
+    }
+
+    /// ------------ ETH AUCTION ------------ ///
+
+    function runETH() public {
+        vm.prank(address(seller));
+        auctions.createAuction(address(token), 0, 1 days, 0.1 ether, payable(address(sellerFundsRecipient)), 1000, address(0), 0);
+
+        vm.warp(1 hours);
+        vm.prank(address(bidder));
+        auctions.createBid{value: 0.1 ether}(address(token), 0, 0.1 ether, address(finder));
+
+        vm.warp(10 hours);
+        vm.prank(address(otherBidder));
+        auctions.createBid{value: 0.5 ether}(address(token), 0, 0.5 ether, address(finder));
+
+        vm.warp(1 days);
+        vm.prank(address(bidder));
+        auctions.createBid{value: 1 ether}(address(token), 0, 1 ether, address(finder));
+
+        vm.warp(1 days + 1 hours);
+        auctions.settleAuction(address(token), 0);
+    }
+
+    function test_ETHIntegration() public {
```

Comment:
The assertions in this function take into account the royalties paid out to the creator and the finder respectively. However, there doesn’t seem to be an assertion that checks the balances if a protocol fee is set at the time the auction is settled:

https://github.com/ourzora/v3/blob/a5b3c77964c48de5d0cb16a33b2efe135662459d/contracts/modules/ReserveAuction/V1/ReserveAuctionV1.sol#L494

Even if no protocol fee were actually set in these test cases (zero), would the team be open to adding one more require to show that is truly the case, or include a separate set of tests that illustrate what happens when an auction is settled with a non-zero protocol fee? 💭

Reply:
I see your 👍 @kulkarohan -- I'll take some time this week and cut a PR that introduces a test for the protocol fee. 🔪

(If there's a known, hard deadline, do let me know and I can prioritize accordingly.)

Reply:
@kulkarohan - Hi again! 👋

I opened https://github.com/ourzora/v3/pull/140 introducing tests for when the protocol fee is set to 1 basis point (`1 bps` or `0.01%`). I'd love your evaluative review on it so we have complete test coverage for this module. Thanks in advance!